technical-reference/reference-technical-articles/technical-doc-xvs-bridge.md

```diff
@@ -0,0 +1,221 @@
+- `_toAddress`: Receiver address on the Destination Chain
+- `_amount`: Amount of XVS tokens you want to send
+- `_useZro`: `false` (indicating that you are not paying in layer zero ZRO tokens)
+- `_adapterParams`: `0x` (empty bytes as no custom adapter is used)
+
+## 2. Transferring Tokens <a name="transferring-tokens"></a>
+
+The actual token transfer is performed using the `sendFrom` function of the Bridge contract. Follow these steps:
+
+### 2.1. Sending Tokens <a name="sending-tokens"></a>
+
+<figure><img src="../../.gitbook/assets/bnb-dest1.jpg" alt=""><figcaption></figcaption></figure>
+
+1. Call the `sendFrom` function of the Bridge contract with the following parameters:
+   - `_from`: Your address on the BNB
+   - `_dstChainId`: Destination Chain ID (e.g., Sepolia virtual chain ID)
+   - `_toAddress`: The address on the Destination Chain where you want to receive the XVS tokens
+   - `_amount`: Amount of XVS tokens you want to send
+   - `_callParams`: ["RefundGasAddress", "ZROaddress", "adapterParams"]
+     - `RefundGasAddress`: Address where you want to receive a refund for excessive gas sent.
+     - `ZROaddress`: `0x0000000000000000000000000000000000000000` (indicating that you are not paying in ZRO tokens)
+     - `adapterParams`: `0x` (empty bytes, as no custom adapter is used)
+
+## 3. Receiving Tokens on the Destination Chain <a name="receiving-tokens-on-the-remote-chain"></a>
+
+When you send XVS tokens to the Destination Chain using the Bridge, the tokens will be minted by the Bridge contract to the receiver's address on the Destination Chain.
+
+## 4. Transferring Tokens Back to the BNB <a name="transferring-tokens-back-to-the-local-chain"></a>
+
+<figure><img src="../../.gitbook/assets/dest-bnb.jpg" alt=""><figcaption></figcaption></figure>
+
+To transfer XVS tokens back to the BNB, follow a similar process as mentioned in the earlier send section. You don't need to approve the Bridge contract on the Destination Chain to spend XVS tokens on your behalf. The tokens will be burned on the Destination Chain on your behalf and unlocked and transferred to the receiver's address on the BNB.
+
+## 5. Monitoring Transaction Status <a name="monitoring-transaction-status"></a>
+
+After initiating a token transfer, you should wait for the transaction to confirm. This process may take a few minutes. Once the transaction confirms, you will receive the bridged XVS tokens on the Destination Chain.
+
+## 6. Security and Risks <a name="security-and-risks"></a>
+
+<figure><img src="../../.gitbook/assets/risk.jpg" alt=""><figcaption></figcaption></figure>
+
+### 6.1. Ownership Transfer <a name="ownership-transfer"></a>
+
+- Use the `transferOwnership` method in the XVSBridgeAdmin contract to transfer ownership of the admin contract.
+- Use the `transferBridgeOwnership` method to transfer ownership of the Bridge contract from one contract to another.
+- Ownership control is crucial in case of emergencies or security issues.
+
```

Comment:
```suggestion
- The owner of the XVSBridgeAdmin contract will be initially the Guardian, but it will be transferred to Governance as soon as the the Multichain Governance module is deployed.
```